cart.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles/csrt.css">
+</head>
+<body>
+  <nav>
+    <div class="cartPageNavLogo">
+        <img src="assets/84426.svg">
+    </div>
+</nav>
+
+  <div class="cartPage">
+    <div class="cartPageProduct" id="cartPageProduct">
+    </div>
+    <div class="cartPageDetailes">
+     <div class="totalNumber">
+         <button class="totalNumberButton">Total Number</button>
+         <div class="totalNumberResult" id="totalNumberResult"></div>
+     </div>
+       <div class="totalPrice">
+         <button class="totalPriceButton">Total price</button>
+         <div class="totalPriceResult" id="totalPriceResult"></div>
+     </div>
+       <div class="totalPurchase">
+         <button class="totalPurchaseButton" id="totalPurchaseButton">purchase</button>
+     </div>
+    </div>
+ </div>
+ <script src="dom/cart.js"></script>
```

Comment:
I guess you need to load the `functions.js` script before the dom scripts so that you have access to these functions within your dom scripts. 

```suggestion
 </div>
  <script src="utlis/function.js"></script>
 <script src="dom/cart.js"></script>
```